src/utils.ts

```diff
@@ -18,19 +48,42 @@ const coloredFlowLine = (status, colors) => {
     return `${getStatusColor(status)}${status}${CLI_COLORS.DEFAULT}`;
 }
 
-export const printFlowRunsReport = (suiteDescription, suiteFlowRuns, logger, colors) => {
-    if (suiteFlowRuns) {
-        logger.log("");
-        logger.log(`Test Suite [${suiteDescription}] Flow Runs report:`);
-        suiteFlowRuns.map(
-            f => logger.log(`Flow ${f.description} - ${coloredFlowLine(f.status, colors)}`));
-    }
+export const printOnlyFailedFlowRunsReport = (testSuitesRuns, logger, colors) => {
+    if (testSuitesRuns && Array.isArray(testSuitesRuns)) {
+        let total = 0;
+        let totalFailed = 0;
+        let lines: Array<string> =[]
+        testSuitesRuns.forEach(suiteRun => {
+            const { flowRuns } = suiteRun;
+            if (flowRuns && Array.isArray(flowRuns)) {
+                total += flowRuns.length;
+                const suiteLines: Array<string> =[]
+                flowRuns.forEach((f) => {
+                    if (f.status === STATUSES.FAILED) {
+                        suiteLines.push(failedFlowLine(f, colors))
+                    }
+                });
+                if (suiteLines.length > 0) {
+                    lines.push("");
+                    lines.push(`${HALF_TAB}Test Suite ${suiteRun.description} has failed flow:`);
+                    lines = lines.concat(suiteLines);
+                    lines.push(`${HALF_TAB}More info can be found at ${suiteRun.url}`);
+                    totalFailed += suiteLines.length;
+                }
+            }
+        });
+        if (lines.length >0) {
+            logger.log("");
+            logger.log(`Test Plan errors report - ${CLI_COLORS.RED}${totalFailed} flows have failed ${CLI_COLORS.DEFAULT} (out of ${total} total).`)
+            lines.forEach(l=> logger.log(l));
+        }
+}
 }
 
 export const printTestSuitesRunsReport = (testPlanDescription, testSuitesRuns, logger, colors) => {
-    if (testSuitesRuns) {
+    if (testSuitesRuns && Array.isArray(testSuitesRuns)) {
```

Comment:
`&& testSuitesRuns.length`, otherwise we'll print without need the following line:
```
Test Plan [New Test Plan - Mar 31] Suites Runs report:
```